core/src/main/java/de/jplag/normalization/MultipleEdge.java

```diff
@@ -0,0 +1,33 @@
+package de.jplag.normalization;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import de.jplag.semantics.Variable;
+
+class MultipleEdge {
+    private Set<Edge> edges;
+    private boolean isVariableFlow;
+    private boolean isVariableReverseFlow;
+
+    MultipleEdge() {
+        edges = new HashSet<>();
+        isVariableFlow = false;
+    }
+
+    boolean isVariableFlow() {
+        return isVariableFlow;
+    }
+
+    boolean isVariableReverseFlow() {
+        return isVariableReverseFlow;
+    }
+
+    void addEdge(EdgeType type, Variable cause) {
+        if (type == EdgeType.VARIABLE_FLOW)
+            isVariableFlow = true;
+        if (type == EdgeType.VARIABLE_REVERSE_FLOW)
+            isVariableReverseFlow = true;
```

Comment:
maybe use new switch-case-statement to ensure completeness if enum will be extended

Reply:
This doesn't make sense in this context, there already are edge types where nothing happens here.